Fanny/fanny/Menu/FNYStatusBar.swift

```diff
@@ -19,15 +19,20 @@ class FNYStatusBar: NSStatusBar {
     override init() {
         super.init()
         statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
-        applyStatusItemIcon()
+        statusItem?.button?.title = "Loading.."
```

Comment:
Not sure if this is necessary. Can you elaborate?

Reply:
I am not sure either, hehe! It does take those first x seconds of the refresh interval before the first number appears, but maybe the icon should rather be displayed during this time? At the same time this might be confusing; if you have a long refresh interval (60 sec) and  change the setting governing if the temperature rather than the icon should be displayed, but then seemingly nothing changes for the first 60 secs and you wonder if the setting was really changed.